content/general/integrations.textile

```diff
@@ -0,0 +1,63 @@
+---
+title: Integrations
+meta_description: "Integrations enable external services to send data to Ably channels, and for Ably events to send their data to external services."
+meta_keywords: "integrations, integration, integrate, stream, external service, webhook, webhooks, functions"
+section: general
+index: 0
+jump_to:
+  - Events via webhooks#webhooks
+  - Continuous streaming#streaming
+---
+
+Integrations enable data to be sent from Ably to external services and for external services to send data to Ably channels.
+
+Ably events can be used to trigger webhook requests to external services such as "IFTTT":https://ifttt.com/ and "Zapier":https://zapier.com/ or invoke functions such as "Amazon Lambda functions":https://aws.amazon.com/lambda/. Ably events can also be streamed to external streaming and queuing services such as "Amazon Kinesis":https://aws.amazon.com/kinesis/ and "Amazon SQS":https://aws.amazon.com/sqs/.
```

Comment:
Think we can skip the 'functions' in Amazon Lambda

Reply:
It's confusing to see 'events' when the rest of the documentation talks in terms of 'messages'. Could you add some context so readers can understand what we mean by 'events' here?